docs/_includes/footer.html

```diff
@@ -0,0 +1,58 @@
+<footer
+  class="site-contentinfo site-contentinfo--interior slds-grid slds-grid_vertical-align-center slds-order_3"
+  role="contentinfo"
+>
+  <div class="slds-col">
+    <nav>
+      <ul
+        class="slds-list_horizontal slds-has-dividers_right slds-has-block-links slds-wrap"
+      >
+        <li class="slds-item">
+          <a
+            href="https://www.salesforce.com/company/legal/sfdc-website-terms-of-service.jsp"
+            >Terms of Service</a
+          >
+        </li>
+        <li class="slds-item">
+          <a href="https://www.salesforce.com/company/privacy/">Privacy</a>
+        </li>
+        <li class="slds-item">
+          <a href="https://www.salesforce.com/company/disclosure/"
+            >Responsible Disclosure</a
+          >
+        </li>
+        <li class="slds-item">
+          <a href="https://trust.salesforce.com/" target="_blank">Trust</a>
+        </li>
+      </ul>
+    </nav>
+    <p>
+      © Copyright 2015-present Salesforce.com, inc.&nbsp;<a
```

Comment:
Why is the copyright year for this 2015?

Reply:
Changed to use current year.